open_event/static/js/jquery/jquery.ellipsis.js

```diff
@@ -0,0 +1,43 @@
+// Borrowed from http://stackoverflow.com/a/1022672/1562480
+(function ellipsisInit($) {
+    $.fn.ellipsis = function () {
+        return this.each(function () {
+            var el = $(this);
+
+            var text = el.attr("data-original-text");
+            el.find(".text").text(text);
+
+            var multiline = true;
+
+            var t = $(this.cloneNode(true))
+                    .hide()
+                    .css("position", "absolute")
+                    .css("overflow", "visible")
+                    .width(multiline ? el.width() : "auto")
+                    .height(multiline ? "auto" : el.height())
+                ;
+
+            el.after(t);
+
+            function height() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650387989/issues/source?bid=3233864&fileBranchId=3324174#l22)